### DISCUSSION TITLE
B站又加了一个不适配夜间模式的东西

### DISCUSSION THREAD

**Discussion by bigbug-dev** [the1812/Bilibili-Evolved#1718] [提问 / Q&A]
![image](https://user-images.githubusercontent.com/62877853/114054789-be606d80-98c2-11eb-9568-8a833d5074a2.png)

**Comment by bigbug-dev** [comment]
![image](https://user-images.githubusercontent.com/62877853/114058377-fa490200-98c5-11eb-8648-47c6e2bef395.png)
视频评论区也是

**Reply by Geeyun-JY3** [reply]
建议集中反馈到这个 issue 下：#1692
